### PR TITLE
Feature/updater

### PR DESCRIPTION
**Pull request by DendoD96**
Dear Frank, 
with reference to #43 i have developed an OPC UA client camel component for the updater. At the moment it is a very simple component that supports anonymous or username/password auth to OPC UA server and allows to read a value from a specific node. I have tested it only with some scalar values (Double and Int). Since the camel OPC UA Milo client returns a DataValue i have developed a json-jackson transformer in order to marshal it into a JSON and be able to query with JSONata. Also this one is a very simple components that use the default objectMapper to do the marshaling. In these days i will work to improve this two components. 
Let me know what you think.
Daniele

### CODE REVIEW

File: basyx.components/basyx.components.updater/basyx.components.updater.transformer.camel-jsonjackson/src/main/java/basyx/components/updater/transformer/cameljsonjackson/configuration/factory/JsonjacksonDefaultConfigurationFactory.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import basyx.components.updater.core.configuration.factory.DataTransformerConfigurationFactory;
+import basyx.components.updater.transformer.cameljsonjackson.configuration.JsonjacksonTransformerConfiguration;
+
+/**
+ * JsonJackson default configuration factory from default path
+ * 
+ * @author Daniele Rossi
+ *
+ */
+public class JsonjacksonDefaultConfigurationFactory extends DataTransformerConfigurationFactory {
```
Comment (FrankSchnicke) [inline basyx.components/basyx.components.updater/basyx.components.updater.transformer.camel-jsonjackson/src/main/java/basyx/components/updater/transformer/cameljsonjackson/configuration/factory/JsonjacksonDefaultConfigurationFactory.java:23]: Please update the name to "JsonJacksonDefaultConfigurationFactory"

**Review by FrankSchnicke — CHANGES_REQUESTED** [review]
Dear Daniele,

many thanks for providing your PR! There are two minor review points in regard to naming, would it be fine with you to address these points? 

Additionally, we're in the process of changing the license of Eclipse BaSyx to MIT License (see https://gitlab.eclipse.org/eclipsefdn/emo-team/emo/-/issues/237). Thus, merging your commit requires you being fine with this change at a later point of time. Would the MIT license also be fine with you? 

Best regards,
Frank

**Review by FrankSchnicke — APPROVED** [review]

**Comment by DendoD96** [comment]
Hello Frank, 
i have updated the classes names. Regarding the license the MIT license is fine for me.

Best regards,
Daniele

**Comment by FrankSchnicke** [comment]
Many thanks again. Your contribution is highly appreciated.

**Pull request merged**